out/production/H-Ray/Main.hs

```diff
@@ -0,0 +1,4 @@
+module Main where
```

Comment:
`out/production` folder shouldn't be in repository, you should add it to `.gitignore` as well.